src/test/java/com/parasoft/findings/utils/results/testableinput/LocationUtilTest.java

```diff
@@ -0,0 +1,90 @@
+package com.parasoft.findings.utils.results.testableinput;
+
+import com.parasoft.findings.utils.results.violations.LocationsException;
+import com.parasoft.findings.utils.results.xml.IXmlTagsAndAttributes;
+import org.junit.jupiter.api.Test;
+import java.util.Properties;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class LocationUtilTest {
+
+    @Test
+    public void testCreateTestableInput_projIdAndResProjPathAttributesInProperties() throws LocationsException {
+        Properties properties = new Properties();
+        properties.put("projId", "project");
```

Comment:
Use attribute variables in `IXmlTagsAndAttributes` for `projId` and `resProjPath`.